src/commands/stats.ts

```diff
@@ -0,0 +1,57 @@
+import { botID } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/module/client.ts";
+import { botCache } from "../../mod.ts";
+import { cache } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/utils/cache.ts";
+import { avatarURL } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/member.ts";
+import { sendMessage } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/channel.ts";
+
+botCache.commands.set(`stats`, {
+  name: `stats`,
+  guildOnly: true,
+  execute: (message, _args, guild) => {
+    let botMember = guild?.members.get(botID);
+
+    if (!botMember) return;
+
+    let memberCount = 0;
+
+    cache.guilds.forEach((guild) => {
+      memberCount += guild.members.size;
+    });
+
+    return sendMessage(message.channel, {
+      embed: {
+        author: {
+          name: `${botMember?.nick || botMember?.user.username} Stats`,
+          icon_url: avatarURL(botMember),
+        },
+        fields: [
+          {
+            name: "Guilds:",
+            value: cache.guilds.size.toString(),
```

Comment:
```suggestion
            value: cache.guilds.size.toLocaleString(),
```

Use `toLocaleString` it will put the large numbers with `,` like `123,456,789` instead of `123456789`.